sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -231,6 +233,7 @@ def _get_compile_kernel_cmd_out(self, kernel):
         command = [
             vxx, "--target", self.vitis_mode,
             "--advanced.param", "compiler.hlsDataflowStrictMode=off",
+            "-O3",
             "--platform", self.xilinx_platform,
             "--temp_dir", self.tmpdir / 'vxx_comp_tmp',
             "--log_dir", self.tmpdir / 'vxx_comp_log',
```

Comment:
By looking at all the various compilation phases I think they would deserve some comments about the recipe.
What we are looking by adding some passes or not.
There is some rationale in the cuisine. :-)